edx_proctoring/tests/test_api.py

```diff
@@ -950,10 +950,12 @@ def test_alias_timed_out(self):
         """
 
         exam_attempt = self._create_started_exam_attempt()
+        timeout_timestamp = datetime.now(pytz.UTC) - timedelta(minutes=1)
```

Comment:
What makes `alias_timeout` true for this test?  Do we need one for when it is false?